docs/reference/cat/nodes.asciidoc

```diff
@@ -24,6 +24,8 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=bytes]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=http-format]
 
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=include-unloaded-segments]
```

Comment:
I think I'd prefer this to be down at the bottom of the list of query parameters, it's a fairly obscure option.